feature-libs/customer-ticketing/components/customer-ticketing-create/customer-ticketing-create-dialog/customer-ticketing-create-dialog.component.ts

```diff
@@ -10,7 +13,18 @@ export class CustomerTicketingCreateDialogComponent
   extends CustomerTicketingDialogComponent
   implements OnInit
 {
+  ticketCategories: Array<Category>;
+  ticketAssociatedObjects: Array<AssociatedObjects>;
+
   ngOnInit(): void {
+    this.customerTicketingFacade.getTicketCategories().subscribe((category) => {
+      this.ticketCategories = category;
+    });
+    this.customerTicketingFacade
+      .getTicketAssociatedObjects()
+      .subscribe((associatedObject) => {
+        this.ticketAssociatedObjects = associatedObject;
+      });
     this.buildForm();
   }
 
```

Comment:
createTicketRequest  --- line 17

you would need to utilize this function to pass to the newly integrated service to use your apis